src/fan.mli

```diff
@@ -49,3 +49,6 @@ val import : hash_size:int -> string -> t
 (** [import ~hash_size buf] decodes [buf] such that
     [import ~hash_size (export t) = t] if [t] was initially created with
     ~hash_size. *)
+
+val clear : unit -> string
```

Comment:
```suggestion
val empty : string
```
I think this isn't clearing anything.

Reply:
You are right, and I actually don't need it.